chassis/core/src/main/java/com/griddynamics/jagger/invoker/CircularExclusiveAccessLoadBalancer.java

```diff
@@ -45,6 +50,12 @@ protected boolean isToCircleAnIteration() {
             throw new IllegalStateException(String.format("Didn't manage to poll the next pair. Timeout %s",
                                                           pollTimeout));
         }
+        
+        long endMillis = System.currentTimeMillis();
+        if (endMillis - startMillis > 1000) {
+            log.warn("It took {} ms to poll the next pair for the load balancer", endMillis - startMillis);
```

Comment:
It took {} ms to poll the next pair for the load balancer. Possible reason: not enough test data for selected load